app/views/spree/checkout/_confirm.html.erb

```diff
@@ -0,0 +1,25 @@
+<fieldset id="order_details" data-hook>
+  <div class="clear"></div>
+  <legend align="center"><%= t('spree.confirm') %></legend>
+  <%= render partial: 'spree/shared/order_details', locals: { order: @order } %>
+</fieldset>
+
+<br />
+
+<div class="form-buttons" data-hook="buttons">
+  <% Spree::Frontend::Config[:require_terms_and_conditions_acceptance].tap do |requires_acceptance| %>
+    <% if requires_acceptance %>
+      <div class="terms_and_conditions" data-hook="terms_and_conditions">
+        <div class="policy"><%= render partial: "spree/checkout/terms_and_conditions" %></div>
+        <%= check_box_tag :accept_terms_and_conditions, 'accepted', false %>
+        <%= label_tag :accept_terms_and_conditions, t('spree.agree_to_terms_of_service') %>
+      </div>
+    <% end %>
+
+    <%= submit_tag t('spree.place_order'),
+      disabled: requires_acceptance,
```

Comment:
Please align methods params so they start at the same column.

Reply:
Already fixed